Documentation/Building.md

```diff
@@ -1,11 +1,64 @@
 # Building and installing Libevent
 ### Jump to:
+- [Prerequisites](#Prerequisites)
 - [Autoconf](#autoconf)
   - [Flags](#autoconf-flags)
 - [Building on Windows](#building-on-windows)
 - [Building on Unix (With CMake)](#building-on-unix-cmake)
 - [CMake Variables](#cmake-variables)
 
+## Prerequisites
+
+### Linux deb-like (ubuntu/debian/...)
+Just install tools using your preferred package manager if you build using autotools:
```

Comment:
How about this (but it requires proper markdown):

libevent can be built using one of the following build systems:

- autotools (preferred)

... steps ...

- cmake

... steps ...